tests/platform_tests/api/test_sfp.py

```diff
@@ -718,9 +718,9 @@ def test_reset(self, request, duthosts, enum_rand_one_per_hwsku_hostname, localh
                 continue
             info_dict = port_index_to_info_dict[sfp_port_idx]
 
-            # only flap interfaces where are CMIS optics,
-            # non-CMIS optics should stay up after sfp_reset(), no need to flap.
-            if "cmis_rev" in info_dict:
+            # If the xcvr supports low-power mode then it needs to be flapped to come
+            # out of low-power mode after sfp_reset()
+            if self.is_xcvr_support_lpmode(info_dict):
```

Comment:
can we do this check `if "cmis_rev" in info_dict or self.is_xcvr_support_lpmode(info_dict):` to make sure this changes does not break other platforms?

Reply:
Yeah good thought, I'll add that.